### DISCUSSION TITLE
Task Assigning

### DISCUSSION THREAD

**Discussion by struckchure** [struckchure/e_commerce_frontend#4] [General]
Please indicate if you've not been assigned to a task :smiley:

**Comment by olatorera** [comment]
ME

**Reply by struckchure** [reply]
Alright, i'll assign you one

**Reply by struckchure** [reply]
@olatorera 
check #8

**Comment by hassana123** [comment]
🤚🏽here

**Reply by struckchure** [reply]
Alright, i'll add for you too

**Reply by struckchure** [reply]
@hassana123 
check #7

**Comment by Gift-01** [comment]
I have not been assigned any task

**Reply by struckchure** [reply]
Will do

**Reply by struckchure** [reply]
@Gift-01 
check #9

**Comment by Jimohax** [comment]
Me

**Reply by struckchure** [reply]
@Jimohax 
check #10

**Comment by EdSimonn** [comment]
Me

**Reply by struckchure** [reply]
@EdSimonn 
check #8 

*NB*: This is urgent, and should be done ASAP
